.env

```diff
@@ -0,0 +1 @@
+POSTGRES_PASSWORD=wbooks-api
```

Comment:
The .env file should never be in the repo, maybe a .env.example, but not the one you use